src/modules/WebglDistort/Module.js

```diff
@@ -0,0 +1,230 @@
+/*
+* Resolves Fisheye Effect
+*/
+module.exports = function DoNothing(options, UI) {
+
+    var defaults = require('./../../util/getDefaults.js')(require('./info.json'));
+
+    var output;
+    var fx = require('./glfx')
+
+    var dataURItoBlob = function dataURItoBlob(dataURI) {
+        // convert base64/URLEncoded data component to raw binary data held in a string
+        var byteString;
+        if (dataURI.split(',')[0].indexOf('base64') >= 0)
+            byteString = atob(dataURI.split(',')[1]);
+        else
+            byteString = unescape(dataURI.split(',')[1]);
+
+        // separate out the mime component
+        var mimeString = dataURI.split(',')[0].split(':')[1].split(';')[0];
+
+        // write the bytes of the string to a typed array
+        var ia = new Uint8Array(byteString.length);
+        for (var i = 0; i < byteString.length; i++) {
+            ia[i] = byteString.charCodeAt(i);
+        }
+
+        return new Blob([ia], { type: mimeString });
+    }
+
+    var canvasToBlobUrl = function canvasToBlobUrl(canvas) {
+
+        var blob = dataURItoBlob(canvas.toDataURL('image/png'));
+        return window.URL.createObjectURL(blob);
+
+    }
+
+    var warpWebGl = function warpWebGl(id, matrix1, matrix2, download) {
+
+        // try to create a WebGL canvas (will fail if WebGL isn't supported)
+        try {
+            var canvas = fx.canvas(1500, 1500);
+        } catch (e) {
+            alert(e);
```

Comment:
How about an alet that says "Not Supported" instead of just throwing the error. It will mostly be gibberish (i guess).